backend/src/nodes/utils/image_utils.py

```diff
@@ -247,6 +247,6 @@ def preview_encode(img: np.ndarray, target_size: int = 512) -> str:
         f = max(w / target_size, h / target_size)
         img = cv2.resize(img, (int(w / f), int(h / f)), interpolation=cv2.INTER_AREA)
 
-    _, encoded_img = cv2.imencode(".png", (img * 255).astype("uint8"))  # type: ignore
+    _, encoded_img = cv2.imencode(".webp", (img * 255).astype("uint8"))  # type: ignore
```

Comment:
We should decide on the webp PR before merging this

Reply:
These feature as separate, so this PR should not include them. The difference between PNG and WEBP isn't that big anyway, so it's not like this change is required for this PR to work.

Reply:
Right. I cherry picked this commit to make that PR and I just didn't remove it after. I figured we'd have merged it by now